src/main/java/frc/robot/subsystems/drivetrain/SwerveModule.java

```diff
@@ -253,6 +250,17 @@ private void configPID(double kp, double ki, double kd, double kf) {
         angleMotor.config_kF(0, kf, Constants.TALON_TIMEOUT);
     }
 
+
+    private void selectTuneDownMode(boolean isUserDefined) {
+        if (isUserDefined) {
+            driveMotor.configOpenloopRamp(0, Constants.TALON_TIMEOUT);
+            driveMotor.configClosedloopRamp(0, Constants.TALON_TIMEOUT);
+        } else {
+            driveMotor.configOpenloopRamp(Constants.SwerveModule.RAMP_RATE, Constants.TALON_TIMEOUT);
+            driveMotor.configClosedloopRamp(Constants.SwerveModule.RAMP_RATE, Constants.TALON_TIMEOUT);
+        }
```

Comment:
```suggestion
        double ramp;
        if (isUserDefined) {
            ramp = 0;
            } else {
            ramp = Constants.SwerveModule.RAMP_RATE;
        }
        driveMotor.configOpenloopRamp(ramp, Constants.TALON_TIMEOUT);
        driveMotor.configClosedloopRamp(ramp, Constants.TALON_TIMEOUT);
```
Or:
```suggestion
        double ramp = isUserDefined ? 0 : Constants.SwerveModule.RAMP_RATE;
        driveMotor.configOpenloopRamp(ramp, Constants.TALON_TIMEOUT);
        driveMotor.configClosedloopRamp(ramp, Constants.TALON_TIMEOUT);
```
Also, if you don't read the error code returned from the config, there is no need to set the timout.